distribution/conda/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set data = load_setup_py_data() %}
+
+package:
+  name: kotlin-jupyter-kernel
```

Comment:
Cannot get name from `setup.py` because of conda bug: https://github.com/conda/conda-build/issues/1379